Gemfile

```diff
@@ -160,7 +162,7 @@ group :development, :test do
 
   # RSpec testing framework for Ruby on Rails as a drop-in alternative to its default testing
   # framework, Minitest.
-  gem 'rspec-rails', '~> 5.0'
+  gem 'rspec-rails', '> 5.0'
```

Comment:
This is the "pirulo" i mention it might have been deleted (also known as: tide). Why do we want this?